packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -19,6 +19,10 @@ async function waitForAnyText(
     return text !== '';
   });
 }
+const initialAggregationToolbarValue =
```

Comment:
Do you think you can cherry pick e2e test changes to a separate PR / postpone them a bit? I'm cleaning up feature flags and updating the e2e and this already had conflicts with the explain plan work that I had to resolve and I would really prefer to avoid another conflict resolution or being blocked on this PR landing. It's okay if it's not possible, I'll just postpone feature flag removal and work on something else, just tell me what your preference would be here